app/layout.tsx

```diff
@@ -16,6 +18,9 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
+      <head>
+        <link href={designStyles} type="text/css" />
```

Comment:
@bladicito There is a hydration issue because of this link. The issue is probably that this is not a client component. Perhaps you need to wrapp that link into a client component. Could you have a look at it? Here is the error message:
![image](https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/assets/43446100/9bbc3a49-b823-494f-ba8b-6bce2f4e7c9c)
The login and logout is now working on my local machine.

Reply:
This error was already corrected ;-)